rgw/v2/lib/resource_op.py

```diff
@@ -251,6 +251,9 @@ def read(self, ssh_con=None):
             "delete_using_different_user", False
         )
         self.copy_version_object = self.doc["config"].get("copy_version_object", False)
+        self.access_versioned_object = self.doc["config"].get(
+            "access_versioned_object", False
+        )
```

Comment:
@ckulal , this could be test_ops config parameter, instead of config op , something like self.test_ops.get("radoslist_all", False)

Reply:
done